devlab/tests/test_resource_migration.py

```diff
@@ -120,6 +120,35 @@ def test_migrate_nova_security_groups(self):
                                                 resource_name='security_group',
                                                 parameter='description')
 
+    def test_image_members(self):
+
+        def member_list_collector(_images, destination):
```

Comment:
you can simply pass glanceclient ;) this will eliminate the need in `if destination ... else ...`